common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -174,13 +186,15 @@ public ManagedBuffer getBlockData(
       String execId,
       int shuffleId,
       int mapId,
-      int reduceId) {
+      int reduceId,
+      int shuffleGenerationId) {
```

Comment:
nit: I think it's more readable to put `shuffleGenerationId` right after `shuffleId`. We can reorder the `shuffleGenerationId` parameter in other methods as well.

Reply:
Copy that, done in 4c4af33 and also other parameters reorder.